app/screens/main/Main.js

```diff
@@ -143,11 +145,17 @@ class Main extends Component<Props, State> {
         <SideBar src={sideBar} />
         <NavBar>
           <NavBarPart>
-            <Logo src={logo} />
+            <Logo src={logo} onClick={() => this.handleNavigation({ index: 6 })} />
```

Comment:
can it be done like this: onClick={() => shell.openExternal('https://spacemesh.io')}
or should it be done by changing multiple lines of code and disrupting the existing logic (numeration order)?

Reply:
I was using your existing structure. I'll change according to your comment.

Reply:
you didn't cause according to this logic icon nav should be with index 0